microsoft-edge/devtools-guide-chromium/whats-new/2021/05/devtools.md

```diff
@@ -0,0 +1,242 @@
+
+1.  In the **Activity Bar** on the left, select the **Microsoft Edge Tools** button.  In the **Side Bar**, the **Microsoft Edge Tools: Targets** panel appears, showing a single button (**Launch Instance**):
+
+    :::image type="complex" source="../../media/2021/05/edge-tools-extension-in-context-docs-initial-side-bar.msft.png" alt-text="The Microsoft Edge Tools: Targets panel in the Side Bar of Visual Studio Code, showing the Launch Instance button" lightbox="../../media/2021/05/edge-tools-extension-in-context-docs-initial-side-bar.msft.png":::
+       The **Microsoft Edge Tools: Targets** panel in the **Side Bar** of Visual Studio Code, showing the **Launch Instance** button
+    :::image-end:::  
+
+1.  In the **Microsoft Edge Tools: Targets** panel, select the **open a folder** link, and open a folder, such as your own project.  The **Microsoft Edge Tools: Targets** panel now shows two buttons: **Launch Instance** and **Generate launch.json**:
+
+    :::image type="complex" source="../../media/2021/05/edge-tools-extension-in-context-docs-side-bar.msft.png" alt-text="The Microsoft Edge Tools: Targets panel in the Side Bar of Visual Studio Code" lightbox="../../media/2021/05/edge-tools-extension-in-context-docs-side-bar.msft.png":::
+       The **Microsoft Edge Tools: Targets** panel in the **Side Bar** of Visual Studio Code
+    :::image-end:::  
+    
+1.  In the **Side Bar**, select the **Generate launch.json** button.  A `launch.json` file is generated in the root of the project, and is opened for editing.
+
+    :::image type="complex" source="../../media/2021/05/edge-tools-extension-in-context-docs-launch-json-file.msft.png" alt-text="Editing the URL in the launch.json file" lightbox="../../media/2021/05/edge-tools-extension-in-context-docs-launch-json-file.msft.png":::
+       Editing the URL in the `launch.json` file
+    :::image-end:::
+
+1.  In the `launch.json` file, change the URL from an empty string to a URL for your project.  Or leave the string empty, to show the documentation page.
+
+1.  Save the `launch.json` file.
+
+1.  In the **Side Bar**, select the **Launch Project** button.  Microsoft Edge opens, showing the URL that you specified.  Also, in Visual Studio Code, DevTools opens.
+
+For more information, navigate to [Microsoft Edge Developer Tools for Visual Studio Code][VisualstudioMarketplaceMsEdgedevtoolsVscodeEdgeDevtools].
+
+
+<!-- 
+## Announcements from the Chromium project  
+
+[!INCLUDE [contact DevTools team note](../../includes/chromium-whats-new-note.md)]  
+
+### Ipsum et Chromium  
+
+Lorem al lorem et Chromium  To review the history of this feature in the Chromium open-source project, navigate to Issue [xxxxxxx][CRxxxxxxx].  
+
+:::image type="complex" source="../../media/202x/xx/lorem-et-chromium.msft.png" alt-text="Ipsum et Chromium" lightbox="../../media/202x/xx/lorem-et-chromium.msft.png":::
+   Ipsum et Chromium  
+:::image-end:::   
+-->
```

Comment:
Can we remove this template code?

Reply:
plan: remove that template code (the commented-out block).

Reply:
```suggestion
```